engine/src/main/java/com/datatorrent/stram/client/StramClientUtils.java

```diff
@@ -777,4 +777,24 @@ public static String getSocketConnectString(InetSocketAddress socketAddress)
     return host + ":" + socketAddress.getPort();
   }
 
+  public static List<InetSocketAddress> getRMAddresses(Configuration conf)
+  {
+
+    List<InetSocketAddress> rmAddresses = new ArrayList<>();
+    String rmWebAppAddress = conf.get(YarnConfiguration.RM_WEBAPP_ADDRESS);
```

Comment:
No need to get webapp address here as the if, else below provides defaults if webapp address is not available